docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -27,6 +32,9 @@ and activate the ``nextstrain`` conda environment:
 
       conda activate nextstrain
 
+.. |today| date::
```

Comment:
Can this be moved closer to the line defining `|last-month|`?

Reply:
Fixed in 7ea33bf